inputremapper/configs/mapping.py

```diff
@@ -337,28 +374,56 @@ def is_valid(self) -> bool:
         """If the mapping is valid."""
         return True
 
-    @validator("output_symbol", pre=True)
-    def validate_symbol(cls, symbol):
+    @root_validator(pre=True)
+    def validate_symbol(cls, values):
         """Parse a macro to check for syntax errors."""
-        if not symbol:
-            return None
+        symbol = values.get("output_symbol")
+
+        if symbol == "":
+            values["output_symbol"] = None
+            return values
+
+        if symbol is None:
+            return values
 
         symbol = symbol.strip()
+        values["output_symbol"] = symbol
+
+        if symbol == DISABLE_NAME:
+            return values
 
         if is_this_a_macro(symbol):
             try:
                 parse(symbol, verbose=False)  # raises MacroParsingError
-                return symbol
+                return values
             except MacroParsingError as exception:
                 # pydantic only catches ValueError, TypeError, and AssertionError
                 raise ValueError(exception) from exception
```

Comment:
Is this accounted for?

Reply:
Macro errors are displayed correctly and are readable to my satisfaction. Is that what you meant?

Reply:
yeah, I see that you used custom errors everywhere else. So I was wondering why not here.